test_queue_job/tests/test_delay_mocks.py

```diff
@@ -270,6 +270,22 @@ def test_trap_jobs_perform(self):
             self.assertEqual(logs[2].message, "test_trap_jobs_perform graph 3")
             self.assertEqual(logs[3].message, "test_trap_jobs_perform graph 1")
 
+    def test_trap_jobs_prepare_context(self):
```

Comment:
To help others like me, who are not immediately aware of how context is treated in the queue, can you clarify in a docstring that it is checked that only the whitelisted context keys are preserved when running the job?